push-sample-app/README.md

```diff
@@ -14,7 +14,7 @@ This example shows you how to use the [Okta Devices SDK](https://github.com/okta
 ## Prerequisites
 
 In order to use this project, your org's admin needs to:
-1. Add an OIDC app with the proper scopes (`okta.authenticators.manage.self`, `okta.authenticators.read` and `okta.users.read.self`).
+1. Add an OIDC app with the proper scopes (`okta.myAccount.appAuthenticator.manage`, `okta.myAccount.appAuthenticator.read` and `openid`).
```

Comment:
@tommywong-okta what's openid scope in OIDC app? I cannot find it from OKTA API scopes
<img width="741" alt="image" src="https://user-images.githubusercontent.com/32366589/223244710-c42605e6-d7ee-4cc9-b0c4-845bf6d6a6a4.png">

Reply:
It is standard oidc scope not okta api scope

Reply:
So org admin will know what to config to enable the scope openid when comes to "1. Add an OIDC app with the proper scopes (`okta.myAccount.appAuthenticator.manage`, `okta.myAccount.appAuthenticator.read` and `openid`).", right?

Reply:
@IldarAbdullin-okta Since this message is for org admin, should we only include okta api scope here?

Reply:
It is for the dev, not for org admin. Openid is a standard oidc scope which is always enabled for all authz servers

Reply:
@tommywong-okta @IldarAbdullin-okta should we remove openid from here if it is not for org admin?